InfiniteRecharge.iml

```diff
@@ -0,0 +1,333 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file references local files.. May cause issues on other member's laptops? I think it is fine for now, but if we have issues with idea, we should gitignore it